### ISSUE TITLE
Subcortical volume mesaures in the data

### ISSUE THREAD

**Issue by suprsankr** [ramp-kits/autism#16]
Hi
In the anatomical features for parts of the brain in the LH and RH, i can spot the area and thickness measures (on running the starter kit) but it is not clear to me which labels correspond to the volumes of each of these regions. 
The thickness and area labels are labelled like: anatomy_lh_<name of region>_thickness and  anatomy_lh_<name of region>_area for lh and rh. I can see some volume measures with the 'Vol' substring. There are others called 'holes'. So, other than the area and thickness measures, the volume measure seems less in number (Freesurfer says there are 40 subcortical regions). Can someone pl explain how to interpret the volumes and holes measures ?

**Comment by glemaitre** [comment]
@ntraut @r03ert0 you'll be more knowledgeable than me on this topic.

**Comment by suprsankr** [comment]
@glemaitre thanks. i think that was a freesurfer noobie question :) i found this page - http://surfer.nmr.mgh.harvard.edu/fswiki/MorphometryStats?highlight=%28SubCortGrayVol%29
that has some explanation for the labels in freesurfer cortical segmentation. 
also, is there a community for QA on this challenge ?

**Comment by glemaitre** [comment]
We use the issue tracker on GitHub or this [Slack channel](https://ramp-studio.slack.com/shared_invite/MTg1NDUxNDAyNDk2LTE0OTUzOTcwMDQtMThlOWY1NWU0Mg)

**Comment by kegl** [comment]
Yes, use this issue tracker for questions related to the challenge or the data, and the #autism channel on the slack for pinging me if there are any issues with the submission site. We'll monitor both and try to make up the rules as messages happen.

**Comment by suprsankr** [comment]
@glemaitre @kegl told me i could ping you here for a question related to the code in the starter kit. 
the cross_val_score() function call in the starter kit is erring out saying 'list is not a callable object'. How to resolve this ? the scoring parameter is given as: scoring: ['roc_auc', 'accuracy'] which gives an error.

**Comment by glemaitre** [comment]
@suprsankr Did you solve the issue? Please make sure to open an issue which is focus on a certain point. I assume that you solved your issue by updating scikit-learn

**Comment by suprsankr** [comment]
@glemaitre actually no. I updated scikit-learn. 
I just moved on to the implementation for now.
